cv.md

```diff
@@ -0,0 +1,19 @@
+# Oleh Pimkin
+    0987814582, 0322700810
+```sh
+My main goal to become a Senior Software Engineer. Have a good knowledge of software development methodology (Agile, SCRUM), understanding of the testing process (test planning, test design, test cases), software development life cycle and bug life cycle. Also, I am interested in businesss analysis.
+```
+
+ - HTML/CSS
+ - SQL 
+ - GIT 
+ - JIRA
+ 
+Utest (had a lot of projects, mostly e-commerce). 
+Help colleagues to test gambling and dating websites.
+
+- Okten web university - Full Stack Developer 
+- Udemy courses - Software Test Engineer
+##### English courses:
+- Green Gorest, Everbest, KI school, Spacedeer(B2)
+>in general I have been studying English for 5 years
```

Comment:
must not contain commented code, unnecessary files, changes from other branches and generated files** like \*.bundle.js. Please review your changes before contributing. 
.editorconfig, .gitignore, etc. can be included.